app/brave_settings_strings.grdp

```diff
@@ -1263,6 +1263,9 @@
     <message name="IDS_BRAVE_AVATAR_LABEL_89" desc="The label for the avatar_bonbon_ninja.png avatar.">
       Avatar Bonbon Ninja
     </message>
+    <message name="IDS_BRAVE_SHIELDS_EXAMPLE_DIALOG_TEMPLATE" desc="The placeholder for Brave shields add site dialog">
+      example.com
```

Comment:
Do we actually want this translated?

Reply:
it is not translated in chromium 

https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc;l=3006?q=%22%5B*.%5Dexample.com%22&ss=chromium&start=11

removed too